graph_generation/Library/emerald_tablet.md

```
@@ -0,0 +1,497 @@
+
+
+History of the Tablet
+History of the Tablet (largely summarised from Needham 1980, & Holmyard 1957)
+The Tablet probably first appeared in the West in editions of the psuedo-Aristotlean Secretum Secretorum which was actually a translation of the Kitab Sirr al-Asar, a book of advice to kings which was translated into latin by Johannes Hispalensis c. 1140 and by Philip of Tripoli c.1243. Other translations of the Tablet may have been made during the same period by Plato of Tivoli and Hugh of Santalla, perhaps from different sources.
+The date of the Kitab Sirr al-Asar is uncertain, though c.800 has been suggested and it is not clear when the tablet became part of this work.
+Holmyard was the first to find another early arabic version (Ruska found a 12th centruy recension claiming to have been dictated by Sergius of Nablus) in the Kitab Ustuqus al-Uss al-Thani (Second Book of the Elements of Foundation) attributed to Jabir. Shortly after Ruska found another version appended to the Kitab Sirr al-Khaliqa wa San`at al-Tabi`a (Book of the Secret of Creation and the Art of Nature), which is also known as the Kitab Balaniyus al-Hakim fi'l-`Ilal (book of Balinas the wise on the Causes). It has been proposed that this book was written may have been written as early as 650, and was definitely finished by the Caliphate of al-Ma'mun (813-33).
+Scholars have seen similarities between this book and the Syriac Book of Treasures written by Job of Odessa (9th century) and more interestingly the Greek writings of the bishop Nemesius of Emesa in Syria from the mid fourth century. However though this suggests a possible Syriac source, non of these writings contain the tablet.
+Balinas is usually identified with Apollonius of Tyna, but there is little evidence to connect him with the Kitab Balabiyus, and even if there was,the story implies that Balinas found the tablet rather than wrote it, and the recent discoveries of the dead sea scrolls and the nag hamamdi texts suggest that hiding texts in caves is not impossible, even if we did not have the pyramids before us.
+Ruska has suggested an origin further east, and Needham has proposed an origin in China.
+Holmyard, Davis and Anon all consider that this Tablet may be one of the earliest of all alchemical works we have that survives.
+It should be remarked that apparantly the Greeks and Egyptians used the termtranslated as `emerald' for emeralds, green granites, "and perhaps green jasper". In medieval times the emerald table of the Gothic kings of Spain, and the Sacro catino- a dish said to have belonged to the Queen of Sheba, to have been used at the last supper, and to be made of emerald, were made of green glass [Steele and Singer: 488].
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)